backend/src/email/services/email-transactional.service.ts

```diff
@@ -0,0 +1,55 @@
+import { EmailTemplateService, EmailService } from '@email/services'
+import { MailToSend } from '@core/interfaces'
+import { loggerWithLabel } from '@core/logger'
+import { TemplateError } from 'postman-templating'
+
+const logger = loggerWithLabel(module)
+
+/**
+ * Sanitizes an email message and sends it.
+ * @throws TemplateError if the body or subject is invalid
+ * @throws Error if the message could not be sent.
+ */
+async function sendMessage({
+  subject,
+  body,
+  from,
+  recipient,
+  replyTo,
+}: {
+  subject: string
+  body: string
+  from: string
+  recipient: string
+  replyTo?: string
+}): Promise<void> {
+  const sanitizedSubject = EmailTemplateService.client.replaceNewLinesAndSanitize(
```

Comment:
Another thing we will have to handle is to check whether the `recipient` is in the `email_blacklist` table. If so, we should not even attempt to send this message as it risk increasing the bounce rate of our SES credentials. 

With that said, SES bounce notifications from single send will not be processed with this current implementation since processing [is skipped](https://github.com/opengovsg/postmangovsg/blob/e73f09603b4a8eea37d11c58aec3921bfb29a13c/backend/src/email/utils/callback/parsers/ses.ts#L115) if there is no message ID. This will need to be addressed.

Reply:
I've addressed these concerns in the latest 2 commits.